frontend/src/components/managePermissions/table/ApplicationAdminTable.vue

```diff
@@ -73,6 +81,15 @@ const deleteAdmin = (admin: FamAppAdminGetResponse) => {
         },
     });
 };
+
+const highlightNewAppAdminAccesRow = (rowData: any) => {
```

Comment:
Hi Jota, sorry I missed this one in the past, but shall we make this a reusable method as well for all three tables? I know we'll have a completely new design later and these might change as well, but maybe for now let's still try to make it reusable? thanks!